Writerside/topics/Accounts.md

```diff
@@ -1,149 +1,83 @@
 # Accounts
 
-The Catena `AccountsService` is responsible for creating, fetching, updating, and deleting accounts and their associated metadata.
+Catena provides an accounts system that is designed for maximum flexiblity. The rest of Catena is designed to not depend on the Account implementation, as a result, you can bring your own account implementation and swap it out for Catena's.
 
 ## How does Catena define an account?
 
-An `Account` is an `Entity` with an `account_id` and a `display_name`.
+The fields on an account are shown below, Catena primarily cares about the `account_id` and `display_name`, with the `auth_role` being the tie in to the authentication system.
```

Comment:
> tie in to the authentication system

... when using the Catena accounts service. (https://github.com/CatenaTools/catena-tools-core/pull/286 hasn't been merged yet but it moves the direct Catena account dependence in auth into the account service so this only applies when using CatenaAccounts. Otherwise something else would need to provide that authorization check.)